src/init/src/index.ts

```diff
@@ -52,25 +52,30 @@ export const init = async ({
 }: InitOptions = {}): Promise<InitFileResults> => {
   const packageJson = new PackageJson()
   const path = resolve(cwd, 'package.json')
+  let existingData: Manifest | undefined
+
   try {
-    packageJson.read(cwd)
-    // will only return here in case the package.json file does not exist
+    existingData = packageJson.read(cwd)
     logger('package.json already exists')
-    return {}
   } catch (err) {
     if (asError(err).message !== 'Could not read package.json file') {
       throw err
     }
-
-    const name = basename(cwd)
-    const data = template({
-      name,
-      author:
-        author ??
-        getAuthorFromGitUser(await getUser().catch(() => undefined)),
-    })
-    const indent = 2
-    packageJson.write(cwd, data, indent)
-    return { manifest: { path, data } }
   }
```

Comment:
I believe we should still return early here to avoid executing the rest of the function. TBH it's surprising that no tests are breaking without currently 🤔 

```suggestion
    return {}
  }
```